code/src/cljs/sixsq/nuvla/ui/ocre/views.cljs

```diff
@@ -0,0 +1,441 @@
+(ns sixsq.nuvla.ui.ocre.views
```

Comment:
This namespace was removed. You are pushing it back.

Reply:
My bad. Done